students/Dcar89/module-2-html/exercises/recipe

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+    <html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>Document</title>
+    </head>
+    <body>
+        
+    </body>
+    </html>
+    <!--malo sam se igrao pozadinom da nije stalno bijela-->
+<style>
+    body{background-color: darkorange;}
+</style>
+    <meta charset="UTF8">
+
+<title>Apple pie recipe</title>
+</head>
+  <body>
+     
+    <h1>Apple pie</h1>
+
+ <img src="apple-pie.jpg" alt=Pie style="width: 400px;height:250px;">
+ <br>
+
+ <strong>Total time</strong>
+    <p><em> 20 min.  Bake: 45 min </em></p>
+
+    <strong>Makes</strong>
+    <p><em>8 servings</em></p>
+
+    <strong>Ingrediants</strong>
+
+    <dl>
```

Comment:
Popis namirnica nije popis termina koje treba opisati, odnosno definirati nekim opisom. Popis namirnica je samo to - popis. Redoslijed namirnica u popisu nije bitan, stoga preporučam koristiti `<ul>` element (simboličku listu).

Što se tiče ove primjene `<dl>` elementa, njegova primjena nije sintaktički ispravna. `<dl>` element mora sadržavati parove `<dt>` (termin) i `<dd>` (definicija) elemenata. Ovdje je košriten samo `<dt>`, što krši standard i čini dokument nevažećim.